Source/WebCore/bridge/objc/objc_class.mm

```diff
@@ -133,7 +133,7 @@ static inline void convertJSMethodNameToObjc(const CString& jsName, JSNameConver
             if ([thisClass respondsToSelector:@selector(webScriptNameForSelector:)])
                 mappedName = [thisClass webScriptNameForSelector:objcMethodSelector];
 
-            if ((mappedName && [mappedName isEqual:methodName.get()]) || !strcmp(objcMethodSelectorName, buffer.data())) {
+            if ((mappedName && [mappedName isEqual:methodName.get()]) || equalSpans(objcMethodSelectorName, buffer.span())) {
```

Comment:
This is not quite right and broke WK1 tests on mac-wk1. Fixing in https://github.com/WebKit/WebKit/pull/40334.